pkg/kubectl/cmd/get/get.go

```diff
@@ -570,7 +570,10 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	w.Flush()
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
-		fmt.Fprintln(o.ErrOut, "No resources found.")
+
+		resourceType := args[0]
```

Comment:
Does this argument always exist when calling this cmd?

Reply:
yes, it's always the resource type, when not provided it gets handled before this code at https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/get/get.go#L272

if the resource-type is combined with a resource-name (eg, "pod/mypod") it also get handled already